pkg/sql/sem/builtins/generator_builtins.go

```diff
@@ -1879,9 +1896,10 @@ func (p *payloadsForTraceGenerator) Close(_ context.Context) {
 }
 
 var showCreateAllTablesGeneratorType = types.String
+var showCreateAllSchemasGeneratorType = types.String
 
 // Phase is used to determine if CREATE statements or ALTER statements
-// are being generated for showCreateAllTables.
+// are being generated for showCreateAllTables or showCreateAllSchemas.
```

Comment:
We don't need to use Phase for `showCreateAllSchemas` only one "phase" which is to output create statements.

Reply:
Fixed